tests/expectations/tests/issue-674-3.rs

```diff
@@ -9,14 +9,11 @@ pub mod root {
     #[allow(unused_imports)]
     use self::super::root;
     #[repr(C)]
-    #[derive(Debug, Copy, Clone)]
+    #[derive(Debug, Default, Copy, Clone)]
     pub struct nsRefPtrHashtable {
         pub _address: u8,
     }
     pub type nsRefPtrHashtable_UserDataType<PtrType> = *mut PtrType;
-    impl Default for nsRefPtrHashtable {
-        fn default() -> Self { unsafe { ::std::mem::zeroed() } }
-    }
```

Comment:
Again, I think this *should* have been deriving all along; it looks like we're fixing bugs here.